packages/webkitgtk_6.rb

```diff
@@ -0,0 +1,179 @@
+class Webkitgtk_6 < Package
+  description 'Web content engine for GTK'
+  homepage 'https://webkitgtk.org'
+  version '2.40.1'
+  license 'LGPL-2+ and BSD-2'
+  compatibility 'x86_64'
+  source_url 'https://webkitgtk.org/releases/webkitgtk-2.40.1.tar.xz'
+  source_sha256 '64e526984f8cd2161ef03ae949af99c002ff333d615e6386b460164a3c1b7ef6'
+
+  binary_url({
+    x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/webkitgtk_6/2.40.1_x86_64/webkitgtk_6-2.40.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    x86_64: '6f1f6309d27d15ba2040a6f0753c9c36e40ae213f9324dcfc4486e613c671bf7'
+  })
+
+  depends_on 'at_spi2_core' # R
+  depends_on 'cairo'
+  # depends_on 'ccache' => :build
+  depends_on 'dav1d'
+  depends_on 'enchant' # R
+  depends_on 'fontconfig'
+  depends_on 'freetype' # R
+  depends_on 'gcc10' => :build if %w[aarch64 armv7l x86_64].include? ARCH
+  depends_on 'gcc' # R
+  depends_on 'gdk_pixbuf' # R
+  depends_on 'glibc' # R
+  depends_on 'glib' # R
+  depends_on 'gobject_introspection' => :build
+  depends_on 'graphene' # R
+  depends_on 'gstreamer' # R
+  depends_on 'gtk3' # R
+  depends_on 'gtk4' # R
+  depends_on 'gtk_doc' => :build
+  depends_on 'harfbuzz' # R
+  depends_on 'hyphen' # R
+  depends_on 'icu4c' # R
+  depends_on 'lcms' # R
+  depends_on 'libavif' # R
+  depends_on 'libdrm' # R
+  depends_on 'libepoxy' # R
+  depends_on 'libgcrypt' # R
+  depends_on 'libglvnd' # R
+  depends_on 'libgpgerror' # R
+  depends_on 'libjpeg' # R
+  depends_on 'libjxl' # R
+  depends_on 'libnotify'
+  depends_on 'libpng' # R
+  depends_on 'libsecret' # R
+  depends_on 'libsoup'
+  depends_on 'libsoup2' # R
+  depends_on 'libtasn1' # R
+  depends_on 'libwebp' # R
+  depends_on 'libwpe' # R
+  depends_on 'libx11' # R
+  depends_on 'libxcomposite' # R
+  depends_on 'libxdamage' # R
+  depends_on 'libxml2' # R
+  depends_on 'libxrender' # R
+  depends_on 'libxslt' # R
+  depends_on 'libxt' # R
+  depends_on 'mesa' # R
+  depends_on 'openjpeg' # R
+  depends_on 'pango' # R
+  depends_on 'py3_gi_docgen' => :build
+  depends_on 'py3_smartypants' => :build
+  depends_on 'sqlite' # R
+  depends_on 'unifdef' => :build
+  depends_on 'valgrind' => :build
+  depends_on 'vulkan_headers' => :build
+  depends_on 'vulkan_icd_loader'
+  depends_on 'wayland' # R
+  depends_on 'woff2' # R
+  depends_on 'wpebackend_fdo' # R
+  depends_on 'zlibpkg' # R
+
+  no_env_options
+
+  def self.patch
+    system "sed -i 's,/usr/bin,/usr/local/bin,g' Source/JavaScriptCore/inspector/scripts/codegen/preprocess.pl"
+    @arch_flags = ''
+    @gcc_ver = ''
+    if ARCH == 'armv7l' || ARCH == 'aarch64'
```

Comment:
Conditional not needed?

Reply:
This is needed for the gcc options for arm only...